docs/ops_guide/repush.md

```diff
@@ -94,8 +100,14 @@ it defaults to false.
 ```
 kafka.input.combiner.enabled = true
 ```
+
+To specify whether to enable TTL for the repush job. This is an experimental functionality that was newly developed. 
```

Comment:
This feature is under development, or development in progress.

Reply:
The feature is almost done and I'd like to do a manual testing before calling it done. Let me hold off this PR. Once I circle back and get the testing finished, I think we then can merge this PR.